sync/librariesio.js

```diff
@@ -208,21 +208,19 @@ catch(e) {
 let libsByRepos = new Set(libs.map(l => l.repo.toLowerCase()));
 
 let newLibs = [];
-let outputCount = 0;
-
-const pages = [ 1, 2 ]; // 200 results should be enough for now.
-for (let page of pages) {
+for (let page = 0; page < 2; page++) { // Each page is 100 results
```

Comment:
Why 2? Also add a . to end of the comment.

Reply:
just changed a loop here, but I'd better to roll it back, to sort out it later